src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (value: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const perPageSelectors: number =
+    total % Number(perPage) ? Math.floor(total / perPage) + 1 : total / perPage;
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={classNames('page-item', {
+            disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={() => onPageChange(currentPage - 1)}
+          >
+            «
+          </a>
+        </li>
+
+        {Array.from({ length: perPageSelectors }, (_, i) => i + 1).map(i => (
```

Comment:
The `Array.from` call can be optimized by directly using the index for mapping, rather than creating an array and then mapping over it. This can improve performance slightly.